compiler/src/test/java/dagger/internal/codegen/GraphValidationTest.java

```diff
@@ -2072,6 +2072,143 @@ public void subcomponentBindingConflictsWithParent() {
   }
 
   @Test
+  public void subcomponentBindingConflictsWithParentWhenSubcomponentPerformsMembersInjection() {
```

Comment:
This was already correct before this PR right? I wonder if we already have coverage for this, though I guess it can't hurt